nautobot/core/templatetags/helpers.py

```diff
@@ -923,7 +923,7 @@ def django_querystring(context, query_dict=None, **kwargs):
         {% django_querystring my_query_dict foo=3 %}
     """
     if query_dict is None:
-        query_dict = context.request.GET
+        query_dict = context["request"].GET
```

Comment:
In theory we shouldn't be doing this... since it's "vendored" from django. It looks like in the 6.x track it is changed yet again, but want to make sure we consider this perspective before going forward with this.

Reply:
Okay, that seems reasonable